sources/packages/web/src/services/UserService.ts

```diff
@@ -1,10 +1,10 @@
 import ApiClient from "./http/ApiClient";
 import {
-  BCeIDDetailsDto,
-  BCeIDAccountsDto,
-  InstitutionUserDetailsDto,
-  InstitutionUserPersistDto,
-} from "../types/contracts/UserContract";
+  BCeIDDetailsAPIOutDTO,
+  BCeIDAccountsAPIOutDTO,
+  InstitutionUserAPIOutDTO,
+  InstitutionUserPersistAPIInDTO,
+} from "@/services/http/dto/User.dto";
```

Comment:
The DTOs must be added to the DTO index and imported from there.

Reply:
ok. I had to rename one of the dtos because there was a name colision.